arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -230,6 +230,54 @@
         }
       }
     },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/migratetoaad": {
+      "post": {
+        "tags": [
+          "ReplicationFabrics"
+        ],
+        "summary": "Migrates the site to AAD.",
+        "description": "The operation to migrate an Azure Site Recovery fabric to AAD.",
+        "operationId": "ReplicationFabrics_MigrateToAad",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "ASR fabric to migrate.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
Just curious, i didn't see 204 begin declared as a response type for any of the other POST LRO in the same swagger? If not intended, please remove it.